webpack.config.js

```diff
@@ -3,98 +3,90 @@ const HtmlWebpackPlugin = require("html-webpack-plugin");
 const CopyPlugin = require('copy-webpack-plugin');
 const webpack = require("webpack")
 
-module.exports = (env) => {
-  return {
+module.exports = {
     entry: "./src/index.tsx",
-    mode: env,
+    mode: process.env.NODE_ENV ? process.env.NODE_ENV : "none",
```

Comment:
This works, but can be simplified to
```suggestion
    mode: process.env.NODE_ENV,
```
as `undefined` will fallback to `"none"`.